### PR TITLE
Spike/visual updates

### PR DESCRIPTION
**Pull request by bogdanadrianmarc**
No issue to link this PR to, but this fixes some visual issues throughout the app. The reports page loads normally, but the contained pie chart takes around 10 seconds to load, and the user had no knowledge of this. This PR adds a message on the page that notifies the user the pie chart is still loading. Besides this, small change to the user's name in the navigation bar and the messages region on the publish page moved to a more natural location

### CODE REVIEW

**Review by joescottdave — APPROVED** [review]
Lgtm.

File: src/Reports.vue [hunk 1]
```diff
@@ -95,6 +98,14 @@ Add new dataset
     created: function () {
       this.fetchData()
     },
+    mounted () {
+      setTimeout(() => {
```
Comment (ijdickinson) [inline src/Reports.vue:102]: Good idea to inform the user that the data is loading. Currently you're using a fixed timeout ... another option would be to tie the loading state to the actual data loading process. In particular, if `fetchData` returned a  promise, you could use the resolve states of the promise to clear the message. Just a thought

**Review by ijdickinson — COMMENTED** [review]
Looks good. I just made one suggestion to consider using a promise on the data fetch to control the loading message, rather than a fixed timeout

**Review by ijdickinson — APPROVED** [review]
lgtm

**Pull request merged**